source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -288,30 +329,32 @@ The SDAM specification defines `when
 <https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#connection-pool-creation>`_
 the driver should create connection pools.
 
-Once a pool is created, if minPoolSize is set, the pool MUST immediately
-begin creating enough connections such that totalConnections >= minPoolSize.
-These connections MUST be created in a non-blocking manner. The driver SHOULD
-set up (connect, handshake and authenticate) these connections in a non-blocking
-manner, such as via the use of a background thread or asynchronous I/O.
+Once a pool is created, if minPoolSize is set, the pool MUST immediately begin
+being "populated" with enough `Connections <#connection>`_ such that
+totalConnections >= minPoolSize. The Pool MUST be "populated" in a non-blocking
```

Comment:
The 'total >= min'  is  a valid requirement but  a weak one. I suggest either adding "The pool SHOULD populate its connections such that available + in use >= min" as a separate sentence or otherwise incorporating this language somehow.

Reply:
Assuming the steps in "populating the pool" are followed, `totalConnections >= minPoolSize` should be a sufficient requirement. To better reinforce this, I've hyperlinked "populated" to the "Populating the Pool" section and also added some wording within that section mentioning that the intention is that it ensures available + in use >= min (phrased a little differently in English though).

Reply:
:+1:  I think this is good now also.